ipad/ViewControllers/Shift/ShiftViewController.swift

```diff
@@ -257,7 +264,12 @@ class ShiftViewController: BaseViewController {
         let spacer = UIBarButtonItem(barButtonSystemItem: .fixedSpace, target: nil, action: nil)
         navigationItem.setRightBarButtonItems([saveButton, spacer, deleteButton], animated: true)
     }
-    
+
+    private func stationRequired(_ station: String) -> Bool {
```

Comment:
This function looks the same as the function on line 135. If they don't need to be duplicated great but this is not blocking the PR merge either way.

Reply:
I'll make it public so both files can access 👍 Agreed it would be better